stats/measure.go

```diff
@@ -24,8 +24,32 @@ type Measure interface {
 	viewsCount() int
 }
 
-// Measurement is the interface for all measurement types. Measurements are
-// required when recording stats.
+// Measurement is the numeric value measured when recording stats. Each measure
+// provides methods to create measurements of their kind. For example, MeasureInt64
+// provides M to convert an int64 into a measurement.
 type Measurement interface {
```

Comment:
So I've been kicking around an idea on how to displace interfaces with unexported types with a better alternative (since they can have confusing interactions with embedding and have the same API evolution issues as interfaces).  What if you defined Measurement in this way:

```go
type Measurement struct {
  m interface{} // one of *Float64Measure or *Int64Measure
  v interface{} // one of int64 or float64
}
```

Largely the same API surface for the application, but then it's more clear that only certain functions can create measurements.

Reply:
There will be more measurement types, I would revise this idea once we have all the types implemented.